src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -2485,15 +2484,6 @@
 			path = DMLogging;
 			sourceTree = "<group>";
 		};
-		358249E925E910AF00BFF303 /* DMLogging */ = {
-			isa = PBXGroup;
-			children = (
-				358249EA25E910AF00BFF303 /* LogDataItem.swift */,
-				358249EB25E910AF00BFF303 /* DMLogsViewController.swift */,
-			);
```

Comment:
I see the files in the branch and in that project. I wonder why this appears here…? 🤷‍♂️

Reply:
It was duplicated with the merge and i removed one of the reference, the other one exists at line 2480